Loading pytorch-gpu/py3/1.4.0
  Loading requirement: cuda/10.1.1 nccl/2.4.8-1-cuda cudnn/10.1-v7.5.1.10
    gcc/8.3.1 openmpi/4.0.2-cuda
+ JobID=16742
+ rsync -avrz --exclude=figs /linkhome/rech/genimt01/uba22to/4DVARNN-DinAE/ /gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742
+ sed -i -e 's|_LAG_|0|g' /gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742/scripts/OSE/config.yml
+ sed -i -e 's|_DOMAIN_|GULFSTREAM|g' /gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742/scripts/OSE/config.yml
+ sed -i -e 's|_LMOD_|False|g' /gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742/scripts/OSE/config.yml
+ sed -i -e 's|_WREGUL_|True|g' /gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742/scripts/OSE/config.yml
+ sed -i -e 's|_PATH_|/gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742/scripts/OSE|g' /gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742/scripts/OSE/launch_OSE.py
+ python -u /gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742/scripts/OSE/launch_OSE.py
/linkhome/rech/genimt01/uba22to/4DVARNN-DinAE/dinae_4dvarnn/mods/import_Datasets_OSE.py:115: RuntimeWarning: invalid value encountered in less
  ss            = np.sum( np.sum( np.sum( x_test < -100 , axis = -1) , axis = -1 ) , axis = -1)
/var/spool/slurmd/job482041/slurm_script: line 37: 65027 Bus error               (core dumped) python -u ${WORK}/4DVARNN-DinAE_${JobID}/scripts/OSE/launch_OSE.py
+ rm -rf /gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742
rm: cannot remove '/gpfswork/rech/yrf/uba22to/4DVARNN-DinAE_16742': Stale file handle
